bibtex-actions.el

```diff
@@ -103,6 +103,31 @@ may be indicated with the same icon but a different face."
     map)
   "Keymap for 'bibtex-actions'.")
 
+;;; Org-cite citation function
+
+(defun bibtex-actions-format-citation-org (keys)
+  "Format org-cite citations for the entries in KEYS.
+
+The full citation syntax is:
+
+  [cite/style:common prefix ;prefix -@key suffix; ... ; common suffix]
+
+Everything is optional, except the brackets, 'cite' and the colon.
+Also the citation must contain at least a key.
+So its minimal form is:
+
+  [cite:@key]
+
+... or its 'suppress author' variant:
+
+  [cite:-@key]"
+  (let* ((prenote  (if bibtex-completion-cite-prompt-for-optional-arguments (read-from-minibuffer "Prenote: ") ""))
+         (postnote (if bibtex-completion-cite-prompt-for-optional-arguments (read-from-minibuffer "Postnote: ") ""))
+         (prenote  (if (string= "" prenote)  "" (concat prenote  " ")))
+         (postnote (if (string= "" postnote) "" (concat ", " postnote))))
+    ;; this is derived from the pandoc syntax function, but this has two-levels of affixes
```

Comment:
UX question: does it make sense to prompt users for affixes on each cite, or is that just going to be annoying?